src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -474,11 +474,20 @@ export function getEnrichmentBarPlotData(data: { [gene: string]: AlterationEnric
     });
 }
 
-export function getGeneListOptions(data: AlterationEnrichmentRow[]): { label: string, genes: string[] }[] {
+export function getGeneListOptions(data: AlterationEnrichmentRow[], includeAlteration?:boolean): { label: string, genes: string[] }[] {
     if (_.isEmpty(data)) {
         return [USER_DEFINED_OPTION];
     }
 
+    if(includeAlteration) {
+        data = _.map(data,datum=>{
+            return {
+                ...datum,
+                hugoGeneSymbol: datum.hugoGeneSymbol + `: ${CNA_TO_ALTERATION[datum.value!]}`
```

Comment:
seems risky to call the propery a hugeGeneSumbol but also include the OQL stuff at the end.

Reply:
had to include oql at the end to fix this issue. should I do it at line 517 in the same function?

Reply:
i think you could call it "option" or optionName since this function is about composing the options.